requirements/dev-requirements.txt

```diff
@@ -1,9 +1,9 @@
-black>=21.7b0
+black==22.1.0
 ipdb==0.13.9
-ipython==7.31.1
+ipython==8.0.1
```

Comment:
I think you should put dependencies like this `ipython>=7.31.1` to pass the Python3.7 linter because `ipython==8.0.1` cannot be installed in Python3.7

Reply:
Good point. I'll change that.

Reply:
Done